doc/segContexts/SegmentationCategoryTypeModifier-SlicerGeneralAnatomy.json

```diff
@@ -2759,6 +3023,60 @@
             ],
             "SNOMEDCTConceptID": "24924006"
           },
+          {
+            "contextGroupName": "Craniofacial Anatomic Regions",
+            "cid": "4028",
+            "CodeMeaning": "Occipital bone",
```

Comment:
add 3dSlicerLabel and 3dSlicerIntegerLabel for the entire occipital bone (https://www.slicer.org/wiki/Slicer3:2010_GenericAnatomyColors -> 151)